src/NServiceBus.Metrics.PerformanceCounters.Tests/NServiceBus.Metrics.PerformanceCounters.approved.cs

```diff
@@ -25,6 +25,5 @@ public class static PerformanceCountersExtensions
     public class PerformanceCountersSettings
     {
         public void EnableSLAPerformanceCounters(System.TimeSpan sla) { }
-        public void UpdateCounterEvery(System.TimeSpan updateInterval) { }
```

Comment:
With that we would need to bump the major. We could also just mark it as obsolete and say that the value is no longer used